layers/+lang/asm/packages.el

```diff
@@ -56,3 +58,9 @@
 (defun asm/post-init-company ()
   (spacemacs|add-company-hook asm-mode)
   (spacemacs|add-company-hook nasm-mode))
+
+(defun asm/post-init-ggtags ()
+  (add-hook 'asm-mode-hook '(lambda () (ggtags-mode 1))))
```

Comment:
Or is it because you want to enable ggtags-mode but not use it to provide information to Eldoc? (which makes sense sometimes)